tools/fuchsia/fuchsia.cipd.yaml

```diff
@@ -0,0 +1,6 @@
+package: flutter/fuchsia
```

Comment:
Rather tahn using a separate .yaml file, can we just include it all on the command line?

Reply:
Can I do it as a follow up PR?